### PR TITLE
homepage copy changes

### PR DESCRIPTION
**Pull request by hxrts**
cleaner PR

### CODE REVIEW

File: themes/rad/layouts/index.html [hunk 1]
```diff
@@ -4,25 +4,25 @@
...
     </div>
     <div id="term">
       <h2>Terminal-first</h2>
       <p>
-        A Radicle project contains a git repository as well as the
-        issues and proposals associated with that repo. All accessible from
-        your terminal.
+        A Radicle project contains a git repository, plus the associated
+        issues and proposals. Access all the conversations
+        around your codebase from the comfort of your command line.
```
Comment (lftherios) [inline themes/rad/layouts/index.html]: I like this one.

File: themes/rad/layouts/index.html [hunk 2]
```diff
@@ -4,25 +4,25 @@
...
     <div id="prog">
       <h2>Programmable</h2>
       <p>
-        Ever wanted to tweak your code collaboration service? With Radicle each 
-        unit of functionality &ndash;<em>a machine</em>&ndash; is its
-        own litte P2P program, written in the Radicle language. You can change
-        them to suit your needs, or create entirely new ones. More on this soon.
+        Each Radicle utility is a little P2P program we call a Radicle <em>machine</em>.
+        Change them to suit your needs, or create entirely new ones.
+        Developer resources coming soon!
```
Comment (lftherios) [inline themes/rad/layouts/index.html]: Also like this one.

File: themes/rad/layouts/index.html [hunk 3]
```diff
@@ -4,25 +4,25 @@
     <div id="peer">
       <h2>Peer–to–peer</h2>
       <p>
-        The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up
-        your own hosting, and no need to trust companies with your data.
+        Open source development is a peer&ndash;to&ndash;peer process.
+        With Radicle you can collaborate directly, without giving your
+        data to third parties or setting up your own server.
```
Comment (lftherios) [inline themes/rad/layouts/index.html]: Here I prefer the one that mentions explicitly IPFS. I don´t get the "open source development is a peer to peer process¨, Feels too vague to me.

File: themes/rad/layouts/index.html (hunk 1 above)
Comment (juliendonck) [reply themes/rad/layouts/index.html]: I would drop the "at the comfort of". Sounds a bit too startup-y.

File: themes/rad/layouts/index.html (hunk 2 above)
Comment (juliendonck) [reply themes/rad/layouts/index.html]: I still feel that with both of them we haven't nailed the fact that this gives you the power to write your own issues app etc. We talk about these programs, but not what they are good for, or why you'd want to build them. Also I think we need to drop P2P here too. We've said it twice already by now.

File: themes/rad/layouts/index.html [hunk 4]
```diff
@@ -98,19 +98,16 @@ <h2>Propose changes</h2>
...
+        Each utility is a replicated state machine with state updates distributed and replicated via IPFS.
+      </p>
+      <p>
+        The Radicle daemon materializes machines from the local event log,
+        allowing you to query a machine's state or submit new expressions.
+      </p>
+      <p>
+        We've also built an git remote helper for IPFS, so you can also host
+        your git repository on the distributed web.
       </p>
```
Comment (juliendonck) [inline themes/rad/layouts/index.html]: I feel that both of these have nothing to do with the illustration that is below. I thought the point here was to explain how the code collaboration stack works from a higher level.

**Comment by hxrts** [comment]
ok, I tried to address everything above with the last commit

File: themes/rad/layouts/index.html [hunk 5]
```diff
@@ -4,25 +4,24 @@
     <div id="peer">
       <h2>Peer–to–peer</h2>
       <p>
-        The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up
-        your own hosting, and no need to trust companies with your data.
+        Radicle a decentralized, collaborative revision control system built on IPFS.
```
Comment (jkarni) [inline themes/rad/layouts/index.html]: `is`? 

I also prefer the "No hassle" phrasing

File: themes/rad/layouts/index.html [hunk 6]
```diff
@@ -4,25 +4,24 @@
...
     </div>
     <div id="term">
       <h2>Terminal-first</h2>
       <p>
-        A Radicle project contains a git repository as well as the
-        issues and proposals associated with that repo. All accessible from
-        your terminal.
+        A Radicle project contains a git repository, plus the associated
+        issues and proposals. Access all the conversations
+        that happen around your codebase from your command line.
```
Comment (jkarni) [inline themes/rad/layouts/index.html]: "Conversation" sounds too loose and metaphorical for me. The takehome from the user testing sessions to me was that we should be as concrete as possible.

File: themes/rad/layouts/index.html [hunk 7]
```diff
@@ -4,25 +4,24 @@
...
     </div>
     <div id="prog">
       <h2>Programmable</h2>
       <p>
-        Ever wanted to tweak your code collaboration service? With Radicle each 
-        unit of functionality &ndash;<em>a machine</em>&ndash; is its
-        own litte P2P program, written in the Radicle language. You can change
-        them to suit your needs, or create entirely new ones. More on this soon.
+        Each Radicle utility is a P2P program we call a Radicle <em>machine</em>.
+        Need a custom issues system? Modify our version to suit your needs, or create an entirely new utility.
```
Comment (jkarni) [inline themes/rad/layouts/index.html]: The connection between the first sentence and the second is a little unclear.

Comment (jkarni) [reply themes/rad/layouts/index.html]: Also, "custom issue system" makes it feel like that's the only thing you can tweak.

File: themes/rad/layouts/index.html (hunk 6 above)
Comment (jkarni) [reply themes/rad/layouts/index.html]: (Also makes it seem like we have chat or email-like functionality.)

File: themes/rad/layouts/index.html [hunk 8]
```diff
@@ -4,25 +4,25 @@
     <div id="peer">
       <h2>Peer–to–peer</h2>
       <p>
-        The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up
-        your own hosting, and no need to trust companies with your data.
+        Radicle is a decentralized, collaborative revision control system built on IPFS.
```
Comment (jkarni) [inline themes/rad/layouts/index.html]: @geigerzaehler's point is a good one. It is quite misleading to name it a collaborative revision control system. (Also, the only use of that term that I know of is RCS - I think people now use VCS?)

**Pull request merged**